cmd/protoc-gen-alias/README.md

```diff
@@ -0,0 +1,45 @@
+# protoc-gen-jsonshim
+
+`protoc-gen-jsonshim` is a plugin for protoc which generates `MarshalJSON()` and
+`UnmarshalJSON()` functions for `.pb.go` types.  The implementations simply
+redirect to `MarshalJSONPB()` and `UnmarshalJSONPB()` respectively.  This allows
+the types to be used with native Go JSON serialization.
+
+## Usage
+
+Add the executable to your system's PATH, for example:
+
+`$ go install istio.io/tools/cmd/protoc-gen-golang-jsonshim`
```

Comment:
This name seems inconsistent with the protoc-gen-alias path; does this work?

Reply:
Sorry I copy+pasted another plugin and probably forgot the readme entirely. let me update